source/fundamentals/builders/sort.txt

```diff
@@ -261,82 +165,13 @@ in the following order:
    {"_id": 3, "letter": "a", "food": "maple syrup"}
    {"_id": 5, "letter": "a", "food": "milk and cookies"}
 
-Text Search
------------
-
-You can specify the order of the results of a 
-:manual:`text search </text-search/>` by how closely they match your
-search string. Each of your search results has a
-:manual:`text score </reference/operator/aggregation/meta/#exp._S_meta>`, a
-numerical value indicating how well that result matches your search. 
-Use the ``Sorts.metaTextScore()`` static factory method to build your sort
-criteria to sort by the text score.
-
-.. warning:: Make Sure to Create a Text Index
-
-   You need a :manual:`text index </core/index-text/>` on your collection to perform a text search. See the server manual documentation for more
-   information on how to
-   :manual:`create a text index </core/index-text/#create-text-index>`.
```

Comment:
Suggestion: I think "create a text index" should link to the builders > indexes page where it creates a text index

Reply:
Thank you for the note. The [Text Indexes Section](https://docs-mongodbcom-staging.corp.mongodb.com/86d2198/java/docsworker-xlarge/DOCSP-15135-CRUD-Read-Sort-Update/fundamentals/builders/indexes/#text-indexes) does not provide a description of what a text index is used for. My goal in linking to the server documentation is to help readers gain an understanding of how text indexes are used in case they are not familiar with them, and I do not believe the Text indexes section provides that information. Please let me know if you disagree with this assessment.

Reply:
Agreed, but is there a reason you're linking to the same page twice?  I think the first sentence on its own is good enough in that case.

Reply:
One link describes what a text index is, the other describes how to create one. Although they are both on the same page, I think having both gives readers a way to understand what I am describing (what is a text index?) and how to proceed given the situation (how do I make one?).